data_pipeline_api/registry/access_upload.py

```diff
@@ -328,71 +344,77 @@ def upload_model_run(
         data_directory = config_filename.parent / data_directory
     run_id = config["run_id"]
     config_yaml = config["config"]
-    responsible_person = _create_target_data_dict(
-        DataRegistryTarget.users, {DataRegistryField.username: config["responsible_person"]}
-    )
+    accessibility = _get_accessibility(config)
+    namespace = config.get("namespace")
     model_version_str = config_yaml["model_version"]
     model_name = config_yaml["model_name"]
-    accessibility = _create_target_data_dict(
-        DataRegistryTarget.accessibility, {DataRegistryField.name: accessibility_name}
-    )
     open_timestamp = config_yaml["open_timestamp"]
+    model_git_sha = config["metadata"]["git_sha"]
```

Comment:
I presume the code is fine, but you could use some constants for these instead of plain strings?